src/sharing/components/ShareByLink.jsx

```diff
@@ -19,8 +19,8 @@ class ShareByLink extends React.Component {
     }
   }
 
-  copyLinkToClipboard = async () => {
-    await copy(this.props.link)
+  copyLinkToClipboard = () => {
+    copy(this.props.link)
```

Comment:
if(copy(this.props.link)) 
  Alerter.success(`${this.props.documentType}.share.shareByLink.copied`)
else
  Alerter.error ?

Reply:
:+1: